astroid/brain/brain_pathlib.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2022 Deepyaman Datta <deepyaman.datta@utexas.edu>
+
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+
+from typing import Iterator, Optional
+
+from astroid import bases, context, inference_tip
+from astroid.const import PY310_PLUS
+from astroid.exceptions import UseInferenceDefault
+from astroid.manager import AstroidManager
+from astroid.nodes.node_classes import Attribute, Slice, Subscript
```

Comment:
Could you import `from astroid import nodes` here? The `nodes` module shouldn't depend on this brain so we should be fine 😄